rest-api/tests/test_api.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from app import app
+import os
+import tempfile
+
+
+@pytest.fixture
+def client():
+    db_fd, app.config['DATABASE'] = tempfile.mkstemp()
+    app.config['TESTING'] = True
+    client = app.test_client()
+
+    yield client
+
+    os.close(db_fd)
+    os.unlink(app.config['DATABASE'])
+
+
+def test_check_available_url_annotation(client):
+    rv = client.get('/')
+    assert b'/add/annotation' in rv.data
+
+
+def test_check_available_url_creator(client):
+    rv = client.get('/')
+    assert b'/add/creator' in rv.data
+
+
+def test_check_creator_cannot_post(client):
+    rv = client.post('/add/creator', json={
+        'email': 'test@gmail.com'
+    })
+    print(rv.data)
+    assert b'405 Method Not Allowed' in rv.data
+
+
+def test_check_creator_put(client):
+    rv = client.put('/add/creator', data=dict(
+        email='test@gmail.com',
+        home_page='home_page',
+        id='5',
+    ), follow_redirects=True)
+
+    print(rv.data)
+    assert b'created_time":["This field is required."]' in rv.data
```

Comment:
created_time yoksa artık arka tarafta otomatikman, user eklendi date de bugünün date'i olsun saati ile birlikte diye bu field gelmezse ben ekliyorum API'de. O yüzden creator için zorunlu bir field olsa da ön yüzden gelmemesi durumunda artık hata vermeyecek. Arkada biz ekleyeceğiz. Yeni PR'da çıktım bunu.